modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java

```diff
@@ -267,6 +267,8 @@ private ObjectDefinition _toObjectDefinition(
 					}
 				).build();
 				active = objectDefinition.isActive();
+				categorization = objectDefinition.isCategorization();
+				comments = objectDefinition.isComments();
```

Comment:
I analyzed that Portlet property is a boolean and the method used is `portlet = objectDefinition.getPortlet();`. So could you  change issCategorization()/isComments() to the method get, please?

Reply:
Okay. thx